graphql/resolvers.go

```diff
@@ -1064,6 +1067,49 @@ func (r *mutationResolver) SaveSubscription(ctx context.Context, subscription re
 	return true, nil
 }
 
+func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings *restModel.APIUserSettings) (bool, error) {
+	usr := route.MustHaveUser(ctx)
+	adminSettings, err := evergreen.GetConfig()
+	if err != nil {
+		return false, errors.Wrap(err, "Error retrieving Evergreen settings")
```

Comment:
we always return a gql error code for errors

Reply:
whoops forgot this.